geodiff/tests/test_geometry_utils.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ GEODIFF - MIT License
+ Copyright (C) 2020 Martin Dobias
+*/
+
+#include "gtest/gtest.h"
+#include "geodiff_testutils.hpp"
+#include "geodiff.h"
+
+#include "changesetreader.h"
+#include "sqliteutils.h"
+
+#include <iostream>
+
+
+TEST( GeometryUtilsTest, test_wkb_from_geometry )
```

Comment:
we still need python test too (smoke test is ok, just to make sure the python wrapper works)